tests/scripts/all.sh

```diff
@@ -2143,7 +2143,7 @@ component_test_psa_crypto_config_accel_hash_use_psa () {
 
     loc_accel_list="ALG_MD5 ALG_RIPEMD160 ALG_SHA_1 ALG_SHA_224 ALG_SHA_256 ALG_SHA_384 ALG_SHA_512"
     loc_accel_flags=$( echo "$loc_accel_list" | sed 's/[^ ]* */-DLIBTESTDRIVER1_MBEDTLS_PSA_ACCEL_&/g' )
-    make -C tests libtestdriver1.a CFLAGS="$ASAN_CFLAGS $loc_accel_flags" LDFLAGS="$ASAN_CFLAGS"
+    make -C tests libtestdriver1.a CFLAGS="-O2 -Werror $loc_accel_flags"
```

Comment:
I disagree with removing Asan/UBsan from the driver. We aren't testing the driver, sure, but if the core passes an invalid pointer to a driver, having Asan in the driver helps.

Reply:
Ok, good point. I wasn't entirely sure about that, which is why I didn't apply it everywhere before getting feedback. I'll revert it.